.gitignore

```diff
@@ -37,3 +37,5 @@ kuma/static/js/libs/ckeditor/source/ckbuilder
 htmlcov/
 locale/**/*.mo
 /bower_components
+kuma/static/compiled-css/*
```

Comment:
The compiled CSS files should not be compiled into the kuma folder since that's where the assets are. Please create them inside a separate folder that is outside the kuma/ directory (imagine the kuma directory as a non-writable file system). Very soon we'll need to push those files to S3 automatically, so it'd be best to have that build folder in the root directory that we configure with the `STATICFILES_DIRS` setting to be used as a source of files to copy.